### PR TITLE
Dd4.16 content

### PR DESCRIPTION
**Pull request by aevesdocker**
PR includes all content requirements for DD 4.16 release (Dev velocity and POS)

### CODE REVIEW

File: desktop/use-desktop/index.md [hunk 1]
```diff
@@ -29,7 +39,7 @@ To display the Docker menu, right-click on the ![whale menu](../../assets/images
 
 - **Dashboard**. This takes you to the Docker Dashboard.
 - **Sign in/Create Dock Id**
```
Comment (usha-mandya) [inline desktop/use-desktop/index.md:41]: nit: Should this be `Create Docker ID`?

**Review by usha-mandya — APPROVED** [review]
Left a minor comment. Looks good otherwise.

File: desktop/use-desktop/index.md (hunk 1 above)
Comment (aevesdocker) [reply desktop/use-desktop/index.md:41]: Whoops! yes, thanks for spotting

Comment (aevesdocker) [inline desktop/use-desktop/index.md:41]:
```suggestion
- **Sign in/Create Docker ID**
```

**Review by usha-mandya — APPROVED** [review]

**Review by dvdksn — APPROVED** [review]

**Pull request merged**